.pre-commit-config.yaml

```
@@ -29,12 +29,12 @@ repos:
       - id: black
         name: Run Black on Tools/build/check_warnings.py
         files: ^Tools/build/check_warnings.py
-        language_version: python3.12
+        language_version: python3
```

Let's revert these here, and deal with them in https://github.com/python/cpython/pull/126971:
```suggestion
        language_version: python3.12
```